### PR TITLE
Added support for constraint attributes

### PR DESCRIPTION
**Pull request by derrabus**
Since Symfony 5.2, it is possible to annotate constraints using PHP 8.0 attributes. When I turn a constraint annotation into an attribute however, it becomes invisible to NelmioApiDocBundle. With this PR, I'd like to fix this problem.

### CODE REVIEW

File: ModelDescriber/Annotations/SymfonyConstraintAnnotationReader.php [hunk 1]
```diff
@@ -150,4 +151,26 @@ private function applyEnumFromChoiceConstraint(OA\Schema $property, Assert\Choic
...
         $setEnumOnThis->enum = array_values($enumValues);
     }
+
+    private function getPropertyAnnotations(\ReflectionProperty $reflection): \Traversable
+    {
+        if (\PHP_VERSION_ID >= 80000) {
+            foreach ($reflection->getAttributes(Constraint::class, \ReflectionAttribute::IS_INSTANCEOF) as $attribute) {
+                yield $attribute->newInstance();
+            }
+        }
```
Comment (phansys) [inline ModelDescriber/Annotations/SymfonyConstraintAnnotationReader.php:161]: IMO, we could extract these lines to a new private method in order to reuse the same logic in `getPropertyAnnotations()` and `getMethodAnnotations()`.

**Review by phansys — APPROVED** [review]

File: ModelDescriber/Annotations/SymfonyConstraintAnnotationReader.php (hunk 1 above)
Comment (derrabus) [reply ModelDescriber/Annotations/SymfonyConstraintAnnotationReader.php:161]: On second thought, we could have a single method `getAnnotations()` and move the `if` block from `updateProperty()` into the method. I'll try that.

Comment (derrabus) [reply ModelDescriber/Annotations/SymfonyConstraintAnnotationReader.php:160]: Pushed, please have a look.

**Comment by GuilhemN** [comment]
Wonderful, thank you very much @derrabus!

**Pull request merged**